e2e-test/20220705.demo1/README.md

```diff
@@ -0,0 +1,148 @@
+# Demo 1 - July, 5th 2022
+
+This test demonstrates user story **As a data intermediary I want to request the fragmented GIPOD LDES data set.** (VSDSPUB-62).
+
+> **Note**: currently, there is no automated test for this demo.
+
+This LDES client E2E test is similar to the [second demo given on May, 24th](../20220524.demo-2/README.md).
+The same context applies.
+
+To demonstrate the correct working of the LDES client you need to:
+
+1. [Start systems](#start-systems).
+2. [Setup demo](#setup-demo).
+3. [Scenario: replication has not yet started](#TODO)
+4. [Scenario: replication has just started](#TODO)
+5. [Scenario: data set contains 2 fragments](#TODO)
+6. [Scenario: data set contains 3 fragments](#TODO)
+7. [Stop systems](#stop-systems)
+
+## Start systems
+To start the docker containers, you need to:
+* create a [Github personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) (for scope `read:packages`)
+* copy the [`.env.example`](./.env.example) file to `.env.user`
+* fill in the credentials for Apache NiFi and the personal access token
+* build and run the containers, passing your `.env.user` settings by executing in a bash shell:
+```bash
+docker compose --env-file .env.user up
+```
+
+This command will use the [docker-compose.yml](./docker-compose.yml) file found in this directory for building and running the following docker containers:
```

Comment:
Maybe describe her that one has to configure the correct version tags for the Docker image artifacts, or will all the docker compose files of all demos updated with the correct artifact versions at the end of the sprint?  Maybe if one decide to have these artifact tags made configurable in one place that this then also can reference to in this document.